### DISCUSSION TITLE
Alpine Expression Error: selectedRecords is not defined

### DISCUSSION THREAD

**Discussion by luhansalimena** [filamentphp/filament#14030] [Help]
### Package

Form builder

### Package Version

^3.2

### How can we help you?

This occurs when trying to load the RelationManager for a specific relationship. The table is supposed to display related records, and I’ve configured it with some custom columns and bulk actions.

Alpine Expression Error: selectedRecords is not defined

<img width="1470" alt="Screenshot 2024-08-22 at 15 20 35" src="https://github.com/user-attachments/assets/5302deb1-350c-410f-817b-baad2439352a">

I’ve already checked the table configuration, cleared the cache, and updated all dependencies, but the error persists. It seems to be related to how Alpine.js handles the selectedRecords variable.

What could be causing this issue, and how can I resolve it?

```

  <?php

  namespace App\Filament\Resources\ProductsResource\RelationManagers;
  
  use Filament\Forms;
  use Filament\Forms\Form;
  use Filament\Resources\RelationManagers\RelationManager;
  use Filament\Tables;
  use Filament\Tables\Table;
  
  class ProdutoPdvRelationManager extends RelationManager
  {
      protected static string $relationship = 'produtoPdv';
  
      public function form(Form $form): Form
      {
          return $form
              ->schema([
                  Forms\Components\TextInput::make('valor')
                      ->required()
                      ->maxLength(255),
              ]);
      }
  
      public function table(Table $table): Table
      {
          return $table
              ->recordTitleAttribute('id')
              ->columns([
                  Tables\Columns\TextColumn::make('pdv_id'),
              ])
              ->filters([
                  //
              ])
              ->actions([
                  Tables\Actions\EditAction::make(),
                  Tables\Actions\DeleteAction::make(),
              ]);
      }
  }

```

Resource

```
public static function getRelations(): array
    {
        return [
            RelationManagers\ProdutoPdvRelationManager::class,
        ];
    }
```

Relation:

 ```
public function produtoPDV()
    {
        return $this->hasMany('App\ProdutoPDV', 'produto_id');
    }
```

Inverse:

```
public function produto()
    {
        return $this->belongsTo('App\Produto', 'produto_id');
    }
```

**Comment by leandrocfe** [comment]
`php artisan about --only=Filament`
what is the Filament version?

**Reply by luhansalimena** [reply]
v3.2.105

**Comment by adonaisuarez** [comment]
I have a similar issue here. In my case it is a many-to-many relationship between two models. Same version, similar RelationManager code (default code generated by artisan command). I followed the documentation and added a RelationManager with this command:

`php artisan make:filament-relation-manager GroupResource items name --attach`

But, when I access the resource editing screen, the relationship table doesn't work and the console error messages show the same: "Alpine Expression Error: selectedRecords is not defined" and "Uncaught ReferenceError: selectedRecords is not defined".

I don't know what could be causing the problem and would appreciate any help. Thanks.

**Comment by mathieutu** [comment]
Same here, but without any relation, and just trying to access the `{resource}?tableAction=edit&tableActionRecord=1` URL.

I think this should be converted to an issue.
